config/environments/development.rb

```diff
@@ -14,7 +14,7 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+  if Rails.root.join('tmp/caching-dev.txt').exist?
```

Comment:
Is that the correct way? What about Windows Filepaths?

https://stackoverflow.com/questions/47200605/rails-what-is-the-correct-usage-of-rails-root-join-when-pointing-to-tmp-cachin

Reply:
It's the way Rubocop wanted it, but I will have a look

Reply:
[This discussion](https://github.com/rubocop-hq/rubocop-rails/issues/195) states that it does not make any difference because `Pathname` abstracts from the used OS. So I would stick with the official RuboCop recommendation until someone complains about it ^^